src/views/Sale/components/SelfBidList/index.tsx

```diff
@@ -128,13 +128,13 @@ export function SelfBidList({ sale, clearingPrice, bids, isFixed }: SelfBidListP
     return (
       <Flex maxHeight="200px" flexDirection="column" style={{ position: 'relative' }}>
         <Flex flexDirection="row" alignItems="center" marginBottom="8px" padding={isMobile ? '0 8px' : '0 16px'}>
-          <Flex flex={isMobile ? 2.5 : 3}>
+          <Flex flex={isMobile ? 2.5 : 3} justifyContent="center">
```

Comment:
It might make more sense to just create a new styled component for all these.
You can extend Flex and then just add justifyContent as the only difference